### PR TITLE
[SofaBaseCollision] CLEAN CylinderModel

### PR DESCRIPTION
**Pull request by ChristianDuriez**
______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

File: SofaKernel/modules/SofaBaseCollision/CylinderModel.cpp [hunk 1]
```diff
@@ -35,29 +35,25 @@ using namespace sofa::defaulttype;
 using namespace sofa::core::collision;
 using namespace helper;
 
-SOFA_DECL_CLASS(Cylinder)
-
-int RigidCylinderModelClass = core::RegisterObject("Collision model which represents a set of rigid cylinders")
+static int RigidCylinderModelClass = core::RegisterObject("Collision model which represents a set of rigid cylinders")
```
Comment (epernod) [inline SofaKernel/modules/SofaBaseCollision/CylinderModel.cpp:38]: why static?

Comment (damienmarchal) [reply SofaKernel/modules/SofaBaseCollision/CylinderModel.cpp:38]: I don't remember why we did it in this PR (a compiler warning maybe ?).

**Comment by damienmarchal** [comment]
[ci-build][with-scene-tests]

**Comment by guparan** [comment]
Hi @ChristianDuriez could you try renaming all the `_*` Datas to `d_*` ? Would be also a nice clean if not too breaking :+1:

**Comment by epernod** [comment]
[ci-build][with-all-tests]

**Comment by damienmarchal** [comment]
I'm a bit surprised with the test failures;... so i restart 
[ci-build][with-all-tests]

**Pull request merged**